src/main/java/org/csanchez/jenkins/plugins/kubernetes/KubernetesCloud.java

```diff
@@ -1120,7 +1133,7 @@ public FormValidation doCheckDirectConnection(
         private static boolean hasPermission(AccessControlled owner) {
             if (owner instanceof Jenkins) {
                 // Regular cloud
-                return owner.hasPermission(Jenkins.ADMINISTER);
+                return owner.hasPermission(Jenkins.MANAGE);
```

Comment:
@Vlatombe I could not find the reason for a requirement for `Jenkins.ADMINISTER` instead of `Jenkins.MANAGE` here. Seems to me that `Jenkins.MANAGE` is restrictive enough ?

Reply:
Indeed, this line should be reverted.

Reply:
@jglick @Vlatombe after digging a bit about this, the remaining Overall/Administer checks were added in https://github.com/jenkinsci/kubernetes-plugin/pull/1580 but apparently we ought to allow users with Overall/Manage since https://github.com/jenkinsci/kubernetes-plugin/pull/1546.

Reply:
#1546 is about letting Managers manage pod templates, but we don't let them edit other Kubernetes cloud configurations.

Reply:
In general, users with only Manage cannot configure `Jenkins.clouds`.

Reply:
But per https://github.com/jenkinsci/kubernetes-plugin/pull/1546 it looks like we only check on the Manage permissions. The only reason this is still guarded by `Overall/Administer` is because core does not allow non admins to access the Cloud configuration page.
In anyway, I am following your recommendation here, and changing to `Overall/Administer`